constants/src/output.rs

```diff
@@ -0,0 +1,11 @@
+//! Output related.
+
+use crate::macros::monero_definition_link;
+
+/// The minimum amount of blocks a coinbase output is locked for.
+#[doc = monero_definition_link!(a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623, "/src/cryptonote_config.h", 40)]
+pub const COINBASE_LOCK_WINDOW: usize = 60;
+
+/// The minimum amount of blocks an output is locked for.
+#[doc = monero_definition_link!(a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623, "/src/cryptonote_config.h", 49)]
+pub const DEFAULT_LOCK_WINDOW: usize = 10;
```

Comment:
Also used in https://github.com/Cuprate/cuprate/pull/300/commits/7902f0a902f1caece3a1a12e268ef97fe13df59a#diff-bdf64dc01a71e5057249b389c20030fd2f8db7be263cee26318bd976500f075dL112-R114.